### PR TITLE
fix(a11y) Make Dropdown more accessible

### PR DESCRIPTION
**Pull request by tryzniak**
- make it focusable
- allow navigation using arrow keys
- dismiss on escape

This is a rework of #2325

### CODE REVIEW

File: src/components/Navigation/Navigation.jsx [hunk 1]
```diff
@@ -45,8 +45,8 @@ export default class Navigation extends React.Component {
               <Dropdown
                 className="navigation__languages"
                 items={[
-                  { title: 'English', url: 'https://webpack.js.org/' },
-                  { title: '中文', url: 'https://doc.webpack-china.org/' }
+                  { lang: 'en', title: 'English', url: 'https://webpack.js.org/' },
```
Comment (montogeek) [inline src/components/Navigation/Navigation.jsx]: What is the reason of this change?

Comment (tryzniak) [reply src/components/Navigation/Navigation.jsx]: It's my mistake. This shouldn't be there. Thanks for taking a look :)

File: src/components/Navigation/Navigation.jsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ export default class Navigation extends React.Component {
                 className="navigation__languages"
                 items={[
                   { title: 'English', url: 'https://webpack.js.org/' },
-                  { title: '中文', url: 'https://doc.webpack-china.org/' }
+                  { lang: 'zh', title: '中文', url: 'https://doc.webpack-china.org/' }
```
Comment (montogeek) [inline src/components/Navigation/Navigation.jsx]: It is fine to left English language with an empty `lang` attribute?

**Comment by montogeek** [comment]
Thank you!
